File: PolicyManagement.Api/PolicyManagement.Models/Claims/SearchClaimsModel.cs
namespace PolicyManagement.Models.Claims
{
    public class SearchClaimsModel
    {
        public string ControlNumber { get; set; }
        public string CustomerName { get; set; }
        public string MobileNumber { get; set; }
        public int? PosId { get; set; }
        public int? InsuranceCompanyId { get; set; }
        public string PolicyNumber { get; set; }
        public string ClaimsNumber { get; set; }
        public string RegistrationNumber { get; set; }
        public string ClaimsEntryFromDate { get; set; }
        public string ClaimsEntryToDate { get; set; }
        public int Mode { get; set; }
        public bool IsShowAll { get; set; }
    }
}

File: PolicyManagementAngular/src/app/app-entites/models/claims/search-claims-model.ts
export interface ISearchClaimsModel {
    ControlNumber: string;
    CustomerName: string;
    MobileNumber: string;
    PosId?: number;
    InsuranceCompanyId?: number;
    PolicyNumber: string;
    ClaimsNumber: string;
    RegistrationNumber: string;
    ClaimsEntryFromDate: string;
    ClaimsEntryToDate: string;
    Mode: number;
    IsShowAll: boolean;
}